scripts/components/map-layers.component.js

```diff
@@ -118,17 +119,30 @@ export default class {
 
   // TODO: develop info function once is clear how it works
   _onInfo(e) {
-    const target = e && e.currentTarget;
+    const target = e && e.currentTarget; //the information icon
+    const bounds = target.getBoundingClientRect(); //position of the icon
+    const top = bounds.top; // top
+    const left = bounds.left + 30; //left
     const uid = target.closest('.layer-item').getAttribute('data-layer-uid');
+    
+    var node = document.querySelector('.tooltip-layout');
+    node.style.display = 'block';
```

Comment:
Use `is-hidden` class instead of direct styles. The fewer direct styles the better.